### PR TITLE
Closes #305: Redesign of Avatars

### LINKED ISSUE #305: Redesign of user avatars
### What are you proposing?

- Replace circles representing players with new avatar icons
- Add faces to the avatars depending on if they are alive or dead
- Move the nickname of the player to the bottom of the avatar (note if the positioning shown in the hi-fi design does not work, putting the name below the avatar is fine)

### Is this ready to implement?

Yes.

Hi-fi prototype: https://www.figma.com/file/zoB3yUb1s6WonkGmRZcRYR/MAFIA-Assignment-2?node-id=327%3A250

Screenshots of new design:
Screen Shot 2021-04-08 at 7 11 28 PM
Screen Shot 2021-04-08 at 7 12 04 PM

Assets that can be used for this:

alive
dead
685-6851196_person-icon-grey-hd-png-download

### Redesign approval:

- [X] Redesign approved by: @Benjamin-Piper

### PR DESCRIPTION
**Pull request by JChamberlain28**
# Description
This PR fixes #305

Change avatar images to people with faces, rather than 'blobs'. Introduced dead faces for those who are dead. Removed and changed CSS to suit the new image-based players, that are different from the shaded div based 'blobs' that preceded it.

# Demo
image
image

# Type of change:
* New feature (Non-breaking change that adds functionality)

# Checklist:
* Have you merged main into your branch?
* Have you tested your changes to ensure it works as expected and does not break existing functionality?
* If applicable, please ensure sufficient tests are added that is related to the changes.
* Please ensure this PR has a `label`, is linked to an `issue` and is related to a `project`
* If new documentation is required for this change, have you created a new documentation issue that describes the documentation needed?

### CODE REVIEW

File: frontend/src/Components/Player.jsx [hunk 1]
```diff
@@ -100,8 +100,10 @@ export default function Player({ playerName, style, childRef }) {
 
     return (
         <div className={playerStyle} style={style} ref={childRef} onClick={validateOnClick(onClick)}>
+            {isHoveredPlayerDead ? <img src={avatarImageDead} alt="dead player avatar" className={styles.playerImg}/> : <img src={avatarImage} alt="player avatar" className={styles.playerImg}/>}
             <div className={styles.playerText}>
                 <p>{playerName.concat(isHoveredPlayerDead ? ' (DEAD)' : '')}</p>
+                <p>{isPlayer ? ' (YOU)' : ''}</p>
```
Comment (khal745) [inline frontend/src/Components/Player.jsx:106]: This wasn't part of the original design, but I don't mind it, helps make position of character clear.

**Review by khal745 — APPROVED** [review]
LGTM, just one comment about something that technically isnt in the design but I think is worth keeping in or will be easy to remove if design team thinks it isn't necessary. Good work!

File: frontend/src/Components/Player.jsx (hunk 1 above)
Comment (JChamberlain28) [reply frontend/src/Components/Player.jsx:106]: Yeh, I did this as a compromise. The original UI had the player blob a different colour to the rest. As this is no longer done, I instead used text to indicate this.

Comment (khal745) [reply frontend/src/Components/Player.jsx:106]: Sorry, I meant the original hifi design in the linked issue

Comment (JChamberlain28) [reply frontend/src/Components/Player.jsx:106]: I know. The hifi did not have any indication of what player you are. The original UI did, so i carried that feature over in a different way.

**Review by ysway — APPROVED** [review]
Thanks for your hard-working Jack, looks good for me

**Pull request merged**